recipes-extended/packagegroups/packagegroup-bisdn-linux-extra.bb

```diff
@@ -0,0 +1,9 @@
+SUMMARY = "Packagegroups for specifying installable package (sets) for BISDN Linux"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHCKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
+
+inherit packagegroup
+
+# optional packages as dependencies of this package to allow easy building
```

Comment:
In the final version of this file in the PR, there are two sections adding dependencies:

- optional packages as dependencies of this package to allow easy building
- utilities useful for debugging (zstd to extract coredumps)

The second section title is clear, but the first is confusing. Looking at the commit history, the first section title appears to refer to the whole file, but in its final version, that is no longer evident. Also, the first section appears to have become miscellaneous stuff that does not fit any other category (of which there is only one right now, but there might be more). If it ends up being the misc section, shouldn't it be at the end rather than at the beginning of the file?

Reply:
I expect most packages to be in the first part, the generic part. So the debug stuff is more a special case. Therefore I'd like to keep it in that order.

I could also just leave it at one alphabetically sorted list.